backend/src/environment.ts

```diff
@@ -63,13 +67,48 @@ export class Environment {
     this.isElevated = await isElevated()
   }
 
-  get manufactureId() {
+  get manufactureId(): number {
     return this.isHeadless ? MANUFACTURE_ID_HEADLESS : MANUFACTURE_ID_STANDARD
   }
 
   get frontend() {
+    let osVersion = os.release()
+    if (this.isMac) {
+      try {
+        let versionInfo: any = plist.parse(fs.readFileSync('/System/Library/CoreServices/SystemVersion.plist', 'utf8'))
+        console.log(JSON.stringify(versionInfo.ProductVersion))
+        osVersion = versionInfo.ProductVersion
+      } catch {
+        console.log('No System Version File: /System/Library/CoreServices/SystemVersion.plist')
+      }
+    }
+
+    if (this.manufacturerDetails === undefined) {
+      try {
+        let manufactererFile = JSON.parse(fs.readFileSync(path.join(this.adminPath, 'manufacturer.json'), 'utf8'))
+        this.manufacturerDetails = manufactererFile.manufacturer
+      } catch (e) {
+        //catch if file does not exist
+        let version = undefined
+        try {
+          let packageFile = JSON.parse(fs.readFileSync(path.join(process.cwd(), '/package.json'), 'utf8'))
```

Comment:
I don't think this is how you want to do it. Can you instead import the package.json file as a normal import?

```
import pkg from '../package.json'
```

Reply:
Thats how I did it originally.  It did not work.  works in other files I have tried but not this one.  If you find a way to make it work let know.

Reply:
@JamieRuderman He will probably have to do this at the top of the file right?

Reply:
Import did not work at the top of the file.

Reply:
Yea, maybe it's just needs a typescript restart... not sure. Should work.